doc/cmp.txt

```diff
@@ -380,6 +422,81 @@ completion.autocomplete~
   nvim-cmp does not completion automatically but you can still use the manual
   completion though.
 
+                                           *cmp-config.window.completion.border*
+window.completion.border~
+  `string | string[] | nil`
+  Border characters used for the completion popup menu when
+  |experimental.native_menu| is disabled.
+
+                                        *cmp-config.window.completion.scrollbar*
+window.completion.scrollbar~
+  `string | false`
+  A setting defining which character to use for the scrollbar.
+
+  * If an empty string (`""`), |hl-PmenuThumb| will be used.
+  * If `false`, no scrollbar will be rendered.
+  * If |experimental.native_menu| is `true`, this setting will be ignored.
+
+  Default: `""`
+  Recommended: `"║"`
+
+                                     *cmp-config.window.completion.winhighlight*
+window.completion.winhighlight~
+  `number`
+  The highlighting used for the completion menu.
+
+  See: 'winhighlight'
+
+                                           *cmp-config.window.completion.zindex*
+window.completion.zindex~
+  `number`
+  The completion window's zindex.
+
+                                               *cmp-config.window.documentation*
+window.documentation~
+  `false | cmp.DocumentationConfig`.
```

Comment:
I'm not 100% sure but I think the final '.' is causing a bad highlighting of the type:
![image](https://user-images.githubusercontent.com/34305318/149005098-d7b9cca0-fd7f-4304-a67f-a1cf416b492b.png)

instead of
![image](https://user-images.githubusercontent.com/34305318/149005177-2d6f3b15-382f-41b6-8ee5-3b4f5d3988a4.png)